### PR TITLE
changed scripts to support cross-env

### PR DESCRIPTION
**Pull request by AndreEPaul**
to run react-scripts on windows machine

### CODE REVIEW

File: front-end/package.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     "styled-components": "^5.1.0"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "cross-env NODE_ENV=development PORT=3001 react-scripts start",
```
Comment (jonabantao) [inline front-end/package.json]: Can you check in your Windows environment if you really do need to set NODE_ENV? I thought react-scripts would handle that?

Comment (AndreEPaul) [reply front-end/package.json]: Yeah I'll double check

Comment (AndreEPaul) [reply front-end/package.json]: Yeah you're right, worked fine without it. Updated

**Review by jonabantao — APPROVED** [review]

**Comment by AndreEPaul** [comment]
These changes can still be seen in 'Get rec route' pull request

**Pull request closed without merging**